draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -939,26 +939,30 @@ following conditions holds:
 
 ## "Same-site" and "cross-site" Requests  {#same-site-requests}
 
-A request is "same-site" if its target's URI's origin's registrable domain
-is an exact match for the request's client's "site for cookies", or if the
-request has no client. The request is otherwise "cross-site".
+Two origins, A and B, are considered same-site if the following algorithm
+returns true:
 
-For a given request ("request"), the following algorithm returns `same-site` or
-`cross-site`:
+1.  If A and B are both scheme/host/port triples:
```

Comment:
Can we use the same "[tuple origin](https://html.spec.whatwg.org/#concept-origin-tuple)" terminology as HTML here?

Reply:
Hi Mike,
Has there been any changes on the IETF/WHATWG term differences? This same point came up during the PR into cookie inc. https://github.com/mikewest/cookie-incrementalism/pull/3#discussion_r390675247

Reply:
Ha! Ok. I still agree with myself from earlier in the year. :)

Ping @mnot @tfpauly. What would y'all like the working group's policy to be vis a vis these terms? I'd like for us to end up using the same terms with the same definition regardless of venue.

Reply:
Ping @mnot and @tfpauly for the conversation here and below ("URL", etc).